ansible/playbooks/roles/repository/files/download-requirements/redhat-7/requirements.x86_64.txt

```diff
@@ -193,6 +184,10 @@ https://grafana.com/api/dashboards/179/revisions/7/download grafana_dashboard_17
 https://grafana.com/api/dashboards/6663/revisions/1/download grafana_dashboard_6663.json
 # RabbitMQ cluster monitoring (via Prometheus)
 https://grafana.com/api/dashboards/10991/revisions/11/download grafana_dashboard_10991.json
+# OpenSearch
+https://artifacts.opensearch.org/releases/bundle/opensearch/1.2.4/opensearch-1.2.4-linux-x64.tar.gz
```

Comment:
This will have to be added to the new download-requirements: https://github.com/epiphany-platform/epiphany/pull/2926

Reply:
Implemented now.